versions/3.0.md

```diff
@@ -1184,6 +1185,23 @@ Locating a linked resource may be performed by either a `href` or `operationId`.
 In the case of an `operationId`, it must be unique and resolved in the scope of the OAS document.
 Because of the potential for name clashes, consider the `href` syntax as the preferred method for specifications with external references.
 
+When the `url` attribute is provided, the target host may be overridden by the value provided.  For example:
+
+```
+components:
+  links:
+    UserRepositories:
+      operationId: getRepositoriesByOwner
+      url: $responseHeaders.link
+  definitions:
+    user:
+      type: object
+```
+
+will extract the URL from the response header named `link`.  If an absolute URL is provided, it will be used instead of the server / path provided in the target operation.  Parameters _may_ be provided as well.  If a `path` parameter is provided, it shall be ignored unless the `url` value contains a matching location for substitution.  If query parameters are provided in the `url` value, they will be interpreted per the operation definition, and the absence of a required query parameter will result in an invalid request.
+
+When a relative path is provided in the `url` attribute, the host defined in the specification will be used.
```

Comment:
When a relative path is provided in the `url` attribute, the host defined in the specification will be used